app/webpack.config.js

```diff
@@ -0,0 +1,71 @@
+const path = require("path");
+const webpack = require("webpack");
+const CaseSensitivePathsPlugin = require("case-sensitive-paths-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  context: path.resolve(__dirname, "src"),
+  entry: "./index.js",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "dist")
+  },
+  devServer: {
+    contentBase: path.resolve(__dirname, "dist")
```

Comment:
[OPTIONAL FOR NOW] If I enter "http://localhost:8080/123" in URL in dev mode, 404 status code is returned. Your app should respond to all pages with your application, which is essential when you'll start implementing routing